packages/node-bridge/bridge.js

```diff
@@ -83,40 +83,9 @@ function normalizeProxyEvent(event) {
   };
 }
 
-/**
- * @param {import('aws-lambda').APIGatewayProxyEvent} event
- */
-function normalizeAPIGatewayProxyEvent(event) {
-  let bodyBuffer;
-  const { httpMethod: method, path, headers, body } = event;
-
-  if (body) {
-    if (event.isBase64Encoded) {
-      bodyBuffer = Buffer.from(body, 'base64');
-    } else {
-      bodyBuffer = Buffer.from(body);
-    }
-  } else {
-    bodyBuffer = Buffer.alloc(0);
-  }
-
-  return {
-    body: bodyBuffer,
-    headers,
-    isApiGateway: true,
-    method,
-    path,
-    responseCallbackCipher: undefined,
-    responseCallbackCipherIV: undefined,
-    responseCallbackCipherKey: undefined,
-    responseCallbackStream: undefined,
-    responseCallbackUrl: undefined,
-  };
-}
-
 /**
  * @param {import('./types').VercelProxyEvent | import('aws-lambda').APIGatewayProxyEvent} event
- * @return {import('./types').VercelProxyRequest}
+ * @return {import('./types').VercelProxyRequest | undefined }
```

Comment:
maybe should the method throw an error if `event.Action` is `undefined`?

Reply:
Ya I think so